app/views/contributors/index.html.erb

```diff
@@ -1,9 +1,9 @@
-<% title "#{@plan.title} - contributors" %>
+<% title sanitize("#{@plan.title} - contributors") %>
```

Comment:
I see some other instances in the code where we are also translating the text. Such as

```
sanitize(_("#{@plan.title} - contributors")) 
```

We should do the same here.

I believe you had tested this notation was in fact added to translation.io.

Reply:
I'm unsure about this one. Of all the instances of Plan.title I edited, this seems to be the only one that uses the `_()` method for translation.io. Perhaps I should remove the `_()` method from this single case?